tests/PySys/software-management-end-to-end/sm-apt-install-remove-multiple/run.py

```diff
@@ -0,0 +1,108 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate end to end behaviour for the apt plugin
+
+When we install a bunch of packages
+Then they are installed
+When we deinstall them again
+Then they are not installed
+"""
+
+import json
+import requests
+import time
+import sys
+
+sys.path.append("software-management-end-to-end")
+from environment_sm_management import SmManagement
+
+
+class PySysTest(SmManagement):
+    def setup(self):
+        super().setup()
+
+        self.assertThat("False == value", value=self.check_isinstalled("rolldice"))
+
+        self.assertThat("False == value", value=self.check_isinstalled("asciijump"))
+        self.assertThat("False == value", value=self.check_isinstalled("squirrel3"))
+        self.assertThat(
+            "False == value", value=self.check_isinstalled("robotfindskitten")
+        )
+
+    def execute(self):
+
+        action = [
+            {
+                "action": "install",
+                "id": "5475278",
+                "name": "asciijump",
+                "url": " ",
+                "version": "::apt",
+            },
+            {
+                "action": "install",
+                "id": "5473003",
+                "name": "robotfindskitten",
+                "url": " ",
+                "version": "::apt",
+            },
+            {
+                "action": "install",
+                "id": "5474871",
+                "name": "squirrel3",
+                "url": " ",
+                "version": "::apt",
+            },
+        ]
+
+        self.trigger_action_json(action)
+
+        self.wait_until_succcess()
+
+        self.assertThat("False == value", value=self.check_isinstalled("rolldice"))
+
+        self.assertThat("True == value", value=self.check_isinstalled("asciijump"))
+        self.assertThat("True == value", value=self.check_isinstalled("squirrel3"))
+        self.assertThat(
+            "True == value", value=self.check_isinstalled("robotfindskitten")
+        )
+
+        action = [
+            {
+                "action": "delete",
+                "id": "5475278",
```

Comment:
Do those IDs are fake or real ones stored in inventory/managedObject?

Reply:
This IDs are kind of real. I recorded them from the actual calls. They are the ID's of packages in the Software Repository. Thoug, I haven't yet found the place in the REST interface but we can observe them in the ui.